src/include/job.h

```diff
@@ -371,19 +371,20 @@ typedef	struct	hnodent {
 
 typedef struct vmpiprocs {
 	tm_node_id	vn_node;	/* user's vnode identifier */
-	hnodent	       *vn_host;	/* parent (host) nodeent entry */
-	char	       *vn_hname;	/* host name for MPI, if null value */
+	hnodent		*vn_host;	/* parent (host) nodeent entry */
+	char		*vn_hname;	/* host name for MPI, if null value */
 	/* use vn_host->hn_host             */
 	/* host name for MPI */
-	char	       *vn_vname;	/* vnode name */
-	int		vn_cpus;	/* number of cpus allocated to proc */
-	int		vn_mpiprocs;	/* number for mpiprocs  */
-	int		vn_threads;	/* number for OMP_NUM_THREADS */
+	char		*vn_vname;	/* vnode name */
+	int 		vn_cpus;	/* number of cpus allocated to proc */
+	int 		vn_mpiprocs;	/* number for mpiprocs  */
+	int 		vn_threads;	/* number for OMP_NUM_THREADS */
+	long		vn_nppus; /* number of physical proc units */
 	long long	vn_mem;		/* working set size (real mem)	*/
 	long long	vn_vmem;	/* total mem space (virtual)	*/
-	int		vn_naccels;	/* number of accelerators */
-	int		vn_need_accel;	/* should we reserve the accelerators */
-	char	       *vn_accel_model;	/* model of the desired accelerator */
+	int 		vn_naccels;	/* number of accelerators */
+	int 		vn_need_accel;	/* should we reserve the accelerators */
+	char 		*vn_accel_model;	/* model of the desired accelerator */
 	long long	vn_accel_mem;	/* amt of accelerator memory wanted */
 } vmpiprocs;
 
```

Comment:
None of the changes to job.h is related to this bug. We can discard these changes - probably showing due to the way you did the diff for the older changelist